### DISCUSSION TITLE
Want to open the inspector to adjust the properties of a specific table cell (standard.HeaderedRecord) when table cell is clicked.

### DISCUSSION THREAD

**Discussion by goyaljai1** [clientIO/joint#2779] [Q&A] [answered]
### Introduction

I'm working with a table structure where each cell belongs to a standard.HeaderedRecord. I want to open an inspector to adjust the properties of a particular cell when that specific cell is clicked. How can I implement this functionality so that clicking on any table cell triggers the inspector to open and allows me to adjust the properties for that particular cell?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment, marked as answer]
When the user double-clicks the item in the example [DataMapping](https://www.jointjs.com/demos/data-mapping), a dialog with an inspector opens. Please see `itemAction()` function for implementation details.
